includes/multi-currency/class-settings.php

```diff
@@ -382,11 +386,17 @@ public function save() {
 		// Save all settings through the settings API.
 		\WC_Admin_Settings::save_fields( $this->get_settings( $current_section ) );
 
-		// If the manual rate was blank, or zero, we set it to the automatic rate.
-		$manual_rate = get_option( $this->id . '_manual_rate_' . $current_section, false );
-		if ( ! $manual_rate || 0 >= $manual_rate || '' === $manual_rate ) {
-			$available_currencies = $this->multi_currency->get_available_currencies();
-			update_option( $this->id . '_manual_rate_' . $current_section, $available_currencies[ strtoupper( $current_section ) ]->get_rate() );
+		// If we are saving the settings for an individual currency, we have some additional logic.
+		if ( '' !== $current_section ) {
```

Comment:
Soon we'll also introduce the [`store`](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/multi-currency/class-settings.php#L104) section. Do you think we should also include a check for it here in case we forget to do it when enabling it?

Reply:
Good idea! Updated in ccbeab3ff528540b1f30b5eb22a11749a2883f65.

Reply:
Awesome! :shipit: